code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -859,23 +859,23 @@
 	id = "vodka"
 	description = "Number one drink AND fueling choice for Russians worldwide."
 	taste_description = "grain alcohol"
-	color = "#0064C8" // rgb: FALSE, 100, 200
+	color = "#ffffff80" // rgb: FALSE, 100, 200
```

Comment:
rgb comment should be changed to the new rgb

Reply:
worked on a playtest without the rbg

Reply:
yea it's a comment but it should still be changed or just deleted since it's pretty unnecessary